concrete/src/Url/Resolver/Manager/ResolverManager.php

```diff
@@ -18,39 +22,40 @@ class ResolverManager implements ResolverManagerInterface
     protected $default;
 
     /**
-     * @param string               $default_handle
-     * @param URLResolverInterface $default_resolver
+     * @param string $default_handle
+     * @param \Concrete\Core\Url\Resolver\UrlResolverInterface|null $default_resolver
      */
     public function __construct($default_handle = '', UrlResolverInterface $default_resolver = null)
     {
+        $this->priorityTree = [];
```

Comment:
This should just be set on the property
```protected $priorityTree = [];```

That way extending classes that don't call construct won't throw errors

Reply:
Done in 313ebcff9c7a1fdaa36fc841222ab8bb2fa578b7